pkg/karmadactl/cordon.go

```diff
@@ -93,15 +93,8 @@ func NewCmdUncordon(cmdOut io.Writer, karmadaConfig KarmadaConfig, cmdStr string
 
 // CommandCordonOption holds all command options for cordon and uncordon
 type CommandCordonOption struct {
-	// KubeConfig holds the control plane KUBECONFIG file path.
-	KubeConfig string
-
-	// ClusterContext is the name of the cluster context in control plane KUBECONFIG file.
-	// Default value is the current-context.
-	KarmadaContext string
-
-	// DryRun tells if run the command in dry-run mode, without making any server requests.
-	DryRun bool
+	// global flags
+	options.GlobalCommandOptions
```

Comment:
There is a flag `ClusterNamespace` in GlobalCommandOptions that is not used by this command.

I consider whether remove it from GlobalCommandOptions and only remain the most universal flag in GlobalCommandOptions.

Reply:
The #1011 has been merged, shall we rebase this?